notes/0.13.10/clean-leave-roots.markdown

```diff
@@ -0,0 +1,11 @@
+[@pdalpra]: http://github.com/pdalpra
+[106]: https://github.com/sbt/sbt/issues/106
+[2136]: https://github.com/sbt/sbt/pull/2136
+
+### Fixes with compatibility implications
+
+### Improvements
+
+- clean task does not delete root folders anymore, only contents [#106][106]/[#2136][2136] by [@pdalpra][pdalpra]
```

Comment:
Run some tests (running `test:compile` then `clean` on sbt itself) and it seems it has no noticeable impact on the performance of `clean`.
Also, I did not notice other tickets linked to that issue.